src/components/Lunch/ItemLunch.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import shortid from 'shortid'
+
+const ItemLunch = ({ idItem, img, dish, price, lunchs, order, setOrder }) => {
+
+    const updateLunchs = (idItem, price) => {
+        //console.log(idItem);
+        //Items seleccionados
+        const productSelect = lunchs.filter(lun => lun.id === idItem)[0]
+        //console.log(productSelect);
+        //console.log(productSelect.idItem);
```

Comment:
```suggestion
        //console.log(productSelect);
        //console.log(productSelect.idItem);
```
Tratar de borrar los console.log